src/db/userModel.js

```diff
@@ -40,11 +42,21 @@ try {
                 type: String,
                 required: true
             }
-        }]
+        }],
+
+        role: {
```

Comment:
Automatically assigning the 'ADMIN' role based on email can be risky. Consider a more secure role assignment mechanism.